src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -309,39 +319,24 @@ private String getSubscriptionDetails(String newSubscription, String oldSubscrip
                                                          NotificationEventType notificationType) {
-        if (REQUEST_INFO_INCOMPLETE.equals(notificationType)) {
-            if (StringUtils.isEmpty(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())
-                    || "No".equalsIgnoreCase(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())) {
-
-                log.info("Request Incomplete Information with empty or no Information From Appellant for ccdCaseId {}.", notificationWrapper.getNewSscsCaseData().getCcdCaseId());
-                return false;
-            }
+        if (REQUEST_INFO_INCOMPLETE.equals(notificationType)
+            && isNoOrNull(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())) {
+            log.info("Request Incomplete Information with empty or no Information From Appellant for ccdCaseId {}.",
+                notificationWrapper.getNewSscsCaseData().getCcdCaseId());
+            return false;
         }
 
         if (notificationWrapper.getSscsCaseDataWrapper().getState() != null
-                && notificationWrapper.getSscsCaseDataWrapper().getState().equals(State.DORMANT_APPEAL_STATE)) {
-            if (!(NotificationEventType.APPEAL_DORMANT_NOTIFICATION.equals(notificationType)
-                    || NotificationEventType.APPEAL_LAPSED_NOTIFICATION.equals(notificationType)
-                    || NotificationEventType.HMCTS_APPEAL_LAPSED_NOTIFICATION.equals(notificationType)
-                    || NotificationEventType.DWP_APPEAL_LAPSED_NOTIFICATION.equals(notificationType)
-                    || ADMIN_APPEAL_WITHDRAWN.equals(notificationType)
-                    || APPEAL_WITHDRAWN_NOTIFICATION.equals(notificationType)
-                    || STRUCK_OUT.equals(notificationType)
-                    || DECISION_ISSUED.equals(notificationType)
-                    || DIRECTION_ISSUED.equals(notificationType)
-                    || DECISION_ISSUED_WELSH.equals(notificationType)
-                    || DIRECTION_ISSUED_WELSH.equals(notificationType)
-                    || ISSUE_FINAL_DECISION.equals(notificationType)
-                    || ISSUE_FINAL_DECISION_WELSH.equals(notificationType)
-                    || REISSUE_DOCUMENT.equals(notificationType)
-                    || PROVIDE_APPOINTEE_DETAILS.equals(notificationType))) {
-                log.info(format("Cannot complete notification %s as the appeal was dormant for caseId %s.",
-                        notificationType.getId(), notificationWrapper.getCaseId()));
-                return false;
-            }
+            && notificationWrapper.getSscsCaseDataWrapper().getState().equals(State.DORMANT_APPEAL_STATE)
+            && !EVENT_TYPES_NOT_FOR_DORMANT_CASES.contains(notificationType)) {
+            log.info(format("Cannot complete notification %s as the appeal was dormant for caseId %s.",
+                notificationType.getId(), notificationWrapper.getCaseId()));
```

Comment:
```suggestion
            log.info("Cannot complete notification {} as the appeal was dormant for caseId {}.",
                notificationType.getId(), notificationWrapper.getCaseId());
```
Should be able to remove format